clmenu.php

```diff
@@ -43,137 +43,143 @@
+						$6.95
+					</p>
+				</div>
 			</div>
 		</div>
-		<div class="menu">
+		<div class="container-fluid menu">
 			<h2>Other Good Stuff</h2>
-			<div class="menupic">
-				<img src="images/chickensandwich.jpg" alt="chicken wings">
-			</div>
-			<div class="menutext">
-				<p><b>CHICKEN SANDWICH</b><br />
-					Breaded filet on a Kaiser Roll served with Waffle Fries, Honey Mustard and a Pickle. Add Cheese, Tomato and Onion. (Regular or Buffalo Style)<br />
-					$10.95
-				</p>
-			</div>
-			<div class="menupic">
-				<img src="images/breadedshrimp.jpg" alt="chicken wings">
-			</div>
-			<div class="menutext">
-				<p><b>BREADED SHRIMP</b><br />
-					Butterflied and Fried Shrimp served with Waffle Fries and Sweet Chili dipping sauce.<br />
-					$10.95
-				</p>
-			</div>
-			<div class="menupic">
-				<img src="images/chickenwingpizza.jpg" alt="chicken wings">
-			</div>
-			<div class="menutext">
-				<p><b>BAR-Bill CHICKEN WING PIZZA</b><br />
-					Wing Chicken, Mozzarella Cheese and Spicy Blue Cheese sauce.<br />
-					$10.95
-				</p>
-			</div>
-			<div class="menupic">
-				<img src="images/pepperonipizza.jpg" alt="chicken wings">
-			</div>
-			<div class="menutext">
-				<p><b>PEPPERONI PIZZA</b><br />
-					Cheese and Pepperoni (10").<br />
-					$9.95
-				</p>
-			</div>
-			<div class="menupic">
-				<img src="images/fries.jpg" alt="chicken wings">
-			</div>
-			<div class="menutext">
-				<p><b>FRIES</b><br />
-					All battered and seasoned.<br />
-					Choose: Waffle, Shoe String, Beer Battered, or Sweet Potato
-					$3.95
-				</p>
+			<div class="row">
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/chickensandwich.jpg" alt="chicken wings">
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
+					<p><b>CHICKEN SANDWICH</b><br />
+						Breaded filet on a Kaiser Roll served with Waffle Fries, Honey Mustard and a Pickle. Add Cheese, Tomato and Onion. (Regular or Buffalo Style)<br />
+						$10.95
+					</p>
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/breadedshrimp.jpg" alt="chicken wings">
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
+					<p><b>BREADED SHRIMP</b><br />
+						Butterflied and Fried Shrimp served with Waffle Fries and Sweet Chili dipping sauce.<br />
+						$10.95
+					</p>
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/chickenwingpizza.jpg" alt="chicken wings">
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
+					<p><b>BAR-Bill CHICKEN WING PIZZA</b><br />
+						Wing Chicken, Mozzarella Cheese and Spicy Blue Cheese sauce.<br />
+						$10.95
+					</p>
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/pepperonipizza.jpg" alt="chicken wings">
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
```

Comment:
is it necessary to mark sm-xs, and md-lg if they are both the same?

Reply:
Fixed clmenu and eamenu